File: perl/fizzbuzz1.pl
#!/usr/bin/perl
my $i;
for ($i = 1; $i <= 100; $i++) {
	if ($i % 15 == 0) {
		print "Fizzbuzz ";
	} elsif ($i % 5 == 0) {
		print "Buzz ";
	} elsif ($i % 3 == 0) {
		print "Fizz ";
	} else {
		print $i . " ";
	}
}

File: ruby/fizzbuzz1.rb
(1..100).each{|n|
    if n % 15 == 0
        puts "Fizzbuzz"
    elsif n % 5 == 0
        puts "Buzz"
    elsif n % 3 == 0
        puts "Fizz"
    else
        puts n
    end
}
